### ISSUE TITLE
Error #1064 when creating a user on MariaDB version 11.2.3

### ISSUE THREAD

**Issue by Legoman99573** [phpmyadmin/phpmyadmin#18989]
### Describe the bug

When creating a user account in 6.0.0-dev, an issue occurs when creating a database user and causes it to fail.

### To Reproduce

1. Go to 'User Accounts'
2. Click on 'Add user account'
3. Fill out login information and keep it on Native MySQL authentication
4. You can optionally select ``Create database with same name and grant all privileges.`` and/or ``Grant all privileges on wildcard name (username\_%).``
5. Can do whatever with Global Privileges, which has no effect, but have tested with the same results
6. Click on 'Create user'
7. Should get the following error when executed.
`#1064 - You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '*543314E4807C564E2F1E4875839F771CC2187C9D' at line 1`

### Expected behavior

Upon pressing the 'Create user' button, it should have created the user and any other options that were selected.

### Screenshots
Without Global Privileges selected
![image](https://github.com/phpmyadmin/phpmyadmin/assets/7460704/8490f62a-bdc7-4013-83cc-6a5abca4944c)

With Global Privileges selected
![image](https://github.com/phpmyadmin/phpmyadmin/assets/7460704/346e991d-67d2-4d1f-86c7-11b0eacc727f)

### Server configuration

- Operating system: Rocky Linux 9.3 (Blue Onyx)
- Web server: nginx/1.20.1
- Database version: MariaDB 11.2.3
- PHP version: 8.3.2
- phpMyAdmin version: 6.0.0-dev commit 048c51f

### Client configuration

- Browser: Google Chrome Version 121.0.6167.161
- Operating system: Window 11 Version 10.0.22631 Build 22631

### Additional context

Noticed when editing the query, if I do something like this, it will pass no issues:
`CREATE USER 'testuser'@'%' IDENTIFIED VIA mysql_native_password USING PASSWORD('testing-only'); GRANT ALL PRIVILEGES ON *.* TO 'testuser'@'%' REQUIRE NONE WITH GRANT OPTION MAX_QUERIES_PER_HOUR 0 MAX_CONNECTIONS_PER_HOUR 0 MAX_UPDATES_PER_HOUR 0 MAX_USER_CONNECTIONS 0;`

**Comment by liviuconcioiu** [comment]
Can you try this fix https://github.com/phpmyadmin/phpmyadmin/pull/18316/commits/b6667e6174218b89ea9d532e22ca31f9f97e6feb?

I think it's a duplicate of #18210.

**Comment by Legoman99573** [comment]
> Can you try this fix [b6667e6](https://github.com/phpmyadmin/phpmyadmin/commit/b6667e6174218b89ea9d532e22ca31f9f97e6feb)?
> 
> I think it's a duplicate of #18210.

I have tried this and it has not resolved the issue either.

**Comment by johnson361** [comment]
Hello [Legoman99573] can you please merge my commit as this is tested . i have to create a new PR for another issue

**Comment by Legoman99573** [comment]
> Hello [Legoman99573] can you please merge my commit as this is tested . i have to create a new PR for another issue

Already done that ;)

**Comment by johnson361** [comment]
Sorry , this is not merged into master branch sir . Can you please recheck . 
https://github.com/phpmyadmin/phpmyadmin/pull/18996

**Comment by liviuconcioiu** [comment]
> Hello [Legoman99573] can you please merge my commit as this is tested . i have to create a new PR for another issue

You have to create each new PR based on the `QA_5_2` or `master` branch, depending on issue, without merging your commits to the new PR.

> Sorry , this is not merged into master branch sir . Can you please recheck . #18996

@Legoman99573 doesn't have merge rights, only project maintainers. Merging can take between 1 day or a few months.
